tests/functionality.py

```diff
@@ -193,6 +193,10 @@ def _setup(try_revision, library_filter):
     def _cleanup(u, expected_values):
         for l in u.dbProvider.get_libraries():
             u.dbProvider.delete_job(l, expected_values.library_version_id)
+        # Needed so we do a database cleanup and prevent locking in MySQL.
```

Comment:
Should we add a TODO: comment to come back to this?

Reply:
I think I can remove this now...